README.md

```diff
@@ -5,7 +5,6 @@ We are using [nuget-license](https://github.com/tomchavakis/nuget-license) to do
 
 ## Inputs
 see action definition [action.yaml](action.yaml)\
```

Comment:
@datlam162 you should remove the \ as this is shown in thre readme file. i think it was to enforce linebreak but it seems not to work.

Reply:
i fixed this already & also added some missing documentation about the ignorePackages.json as this new feature was undocumented.